pages/CitationStyleLanguageHandler.inc.php

```diff
@@ -94,54 +97,45 @@ public function _setupRequest($args, $request) {
 
 		$this->citationStyle = $args[0];
 		$this->returnJson = isset($userVars['return']) && $userVars['return'] === 'json';
-		$this->article = Services::get('submission')->get($userVars['submissionId']);
-
-		if (!$this->article) {
+		$this->submission = Services::get('submission')->get($userVars['submissionId']);
+		$this->issue = $userVars['issueId'] ? Services::get('issue')->get($userVars['issueId']) : null;
```

Comment:
Cast the IDs to `int`s before using them, just to be safe. This line and the one above.